draft-ietf-rats-corim.md

```diff
@@ -2179,16 +2264,16 @@ Additionally, the creation of Attestation Results is out-of-scope for this docum
 Phase 5: Verifier Augmentation
 
 Claims related to Verifier-applied consistency checks are asserted under the authority of the Verifier.
-For example, the `attest-key-triple-record` may contain a cryptographic key to which the Verifier applies certificate path construction and validation.
-Validation may reveal an expired certificate.
-The Verifier implementation might generate a certificate path validation exception that is handled externally, or it could generate a Claim that the certificate path is invalid.
+For example, the Verifier may supply evidence freshness nonces to the Attester to be included in Evidence.
+If a Verifier nonce is used, the Verifier may augment the ACS with a nonce Claim using Verifier authority.
```

Comment:
What is a nonce Claim? Is this something like but isn't exactly `eat_nonce`? Given that the Verifier is allowed to add anything it wants with its authority, maybe we should drop this sentence and allow the reader to decide what claims they want to add.

Reply:
This is offered as an example. It is already assumed that the reader / implementer will decide what claims they want to add. Since the previous exemplary text is addressed as part of the triples, it isn't a good example.